docs/index.rst

```diff
@@ -0,0 +1,112 @@
+.. ##
+.. ## Copyright (c) 2017-23, Lawrence Livermore National Security, LLC
+.. ## and RAJA Performance Suite project contributors. 
+.. ## See the RAJAPerf/LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+
+########################
+RAJA Performance Suite
+########################
+
+=============================
+Motivation and Background
+=============================
+
+The RAJA Performance Suite is designed to explore performance of loop-based
+computational kernels found in HPC applications. Specifically, it is
+used to assess and monitor runtime performance of kernels implemented using
+`RAJA C++ performance portability abstractions <https://github.com/LLNL/RAJA>`_.
+The Suite contains a variety of kernels implemented using common parallel 
+programming models, such as OpenMP and CUDA. Some important 
+terminology used in the Suite implementation and discussion includes:
+
+  * **Kernel** is a distinct loop-based computation that appears in the Suite in
+    multiple variants (or implementations), each of which performs the same
+    computation.
+  * **Group** is a subset of kernels in the Suite that originated from the 
+    same source, such as a specific benchmark suite.
+  * **Variant** refers to implementations of Suite kernels that share the same 
+    implementation approach and programming model, such as *baseline OpenMP*, 
+    *RAJA OpenMP*, etc.
+  * **Tuning** refers to an implementation of kernels with a particular 
+    execution parameterization, such as GPU thread-block size 128, GPU 
+    thread-block size 256, etc.
+
+The kernels in the Suite originate from various HPC benchmark suites and 
+applications. For example, the "Stream" group contains kernels from the Babel 
+Stream benchmark, the "Apps" group contains kernels extracted from
+real scientific computing applications, and so forth. Each kernel in the Suite 
+appears in multiple RAJA and non-RAJA (i.e., *baseline*) variants that use 
+parallel programming models supported by RAJA. Some kernels have multiple 
+tunings of a variant to explore the performance implications of options that 
+a programming model supports.
+
+.. note:: Available variants for a kernel do not need to include all possible
+          variants in the Suite. In some cases, a kernel appears only in the 
+          subset of variants that makes sense for the particular kernel.
```

Comment:
awkward

Reply:
I think I cut and pasted that from the original README docs. I will push a fix.